modules/axfs/src/root.rs

```diff
@@ -131,6 +131,92 @@ impl VfsNodeOps for RootDirectory {
             }
         })
     }
+
+    fn rename(&self, src_path: &str, dst_path: &str) -> VfsResult {
+        self.lookup_mounted_fs(src_path, |fs, rest_path| {
+            if rest_path.is_empty() {
+                ax_err!(NotFound)
+            } else {
+                fs.root_dir().rename(rest_path, dst_path)
+            }
+        })
+    }
+}
+
+#[cfg(feature = "devfs")]
+fn mount_devfs() -> fs::devfs::DeviceFileSystem {
```

Comment:
Move these 4 `mount_*fs()` functions into a new file `mounts.rs`, and rename to `mounts::devfs()`, `mounts::ramfs()`, ...